Makefile

```diff
@@ -0,0 +1,16 @@
+VIRTUAL_ENV ?= .env
+HDF_FILES := $(shell find 'netz/game-files' | sed 's/\.pgn$$/.hdf5/g' | grep '\.hdf5$$' | uniq )
```

Comment:
I think this whole file is a great idea! But why do you do this command this way and not just
`find 'netz/game-files' -regex '.*hdf5'` ?

Reply:
Some of the files may not exist yet. They still have to be created from the corresponding `.pgn` files.

Reply:
Ok so `make` reads here that some `HDF5` files are supposed to be here for making `model.tfc`, but they aren't, so it proceeds to the next rule, got it, ...